### PR TITLE
Add PRLabels to CODEOWNERS

### PR DESCRIPTION
**Pull request by liurunliang**
Init - Add PRLabel to CODEOWNERS

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [ ] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [x] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [x] No refresh required for updates in this PR

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [ ] Check this box if any of the following appy to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [x] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,88 +1,250 @@
...
 /specification/policyinsights/ @pilor
+
+# PRLabel: %PostgreSQL
 /specification/postgresql/ @qingqingyuan
+
 /specification/powerbidedicated/ @tarostok
+
+# PRLabel: %PostgreSQL
 /specification/provisioningservices/ @kvish
+
```
Comment (mikekistler) [inline CODEOWNERS:173]:
```suggestion

# PRLabel: %Purview
/specification/purview/ @mikekistler

```

File: CODEOWNERS [hunk 2]
```diff
@@ -1,88 +1,250 @@
...
 /specification/datalake-analytics/ @ro-joowan
+
+# PRLabel: %Data Lake Store
 /specification/datalake-store/ @ro-joowan
+
+# PRLabel: %Data Migration
 /specification/datamigration/ @hitenjava
+
 /specification/deploymentmanager/ @netrock
+
```
Comment (mikekistler) [inline CODEOWNERS:90]:
```suggestion

# PRLabel: %Device Update
/specification/deviceupdate/data-plane/ @mikekistler
```

**Review by mikekistler — APPROVED** [review]
Looks good! 👍 

I added a few entries.  If this is what we need to get labels onto PRs, let's not wait for a complete list.  Let's get started with what we have and then fill in later where needed.

File: CODEOWNERS [hunk 3]
```diff
@@ -1,88 +1,250 @@
...
+# PRLabel: %Intune
 /specification/intune/ @vrmurthy01
+
+# PRLabel: %IotHub
 /specification/iothub/ @rkmanda
+
+# PRLabel: %KeyVault
 /specification/keyvault/ @heaths @randallilama @jlichwa
+
```
Comment (mikekistler) [inline CODEOWNERS:124]: Found one more:
```suggestion

% PRLabel: %Load Test Service
/specification/loadtestservice/data-plane/ @mikekistler

```

```

**Comment by mikekistler** [comment]
I also have a question on this: is it possible to add an entry to CODEOWNERS that associates a label with a particular directory but NOT a code owner?  I think that would be useful.

**Comment by liurunliang** [comment]
Hi @mikekistler , I have tried it with [CreateRuleFabricBot](https://github.com/Azure/azure-sdk-tools/tree/main/tools/CreateRuleFabricBot). The bot will ignore the entry without code owner.

File: CODEOWNERS (hunk 2 above)
Comment (weidongxu-microsoft) [reply CODEOWNERS:90]: For data-plane, you can use "@Azure/api-stewardship-board" so reviewer is shared to the team (if you do not intend to review for all deviceupdate changes).

File: CODEOWNERS (hunk 1 above)
Comment (weidongxu-microsoft) [reply CODEOWNERS:173]: Should it be `/specification/purview/data-plane/` ?

**Review by weidongxu-microsoft — APPROVED** [review]
Nice work!

Does our bot supports it (or it is a Github Action)?

**Comment by liurunliang** [comment]
Hi @weidongxu-microsoft, Yes, [Fabric Bot](https://github.com/microsoftfabricbot) supports it.

**Comment by weshaggard** [comment]
@liurunliang this is step one of configuration the data then someone has to run the tool to generate the fabricbot configu from this information. Is that something you plan to do separately? Perhaps we can do it in the same PR?

**Review by jsquire — APPROVED** [review]
The general format looks good.   Wes' comment above is an important one.   This forms the basis that the rules can be generated from, but tooling will need to be run to extract the rules and merge them into the bot configuration in the repository.

**Review by tomkerkhove — APPROVED** [review]
Thanks a ton!

**Review by ruowan — APPROVED** [review]

File: CODEOWNERS [hunk 4]
```diff
@@ -1,88 +1,255 @@
...
 /specification/adp/ @Azure/adp
+
+# PRLabel: %Analysis Services
 /specification/analysisservices/ @taiwu
+
+# PRLabel: %Alerts Management
 /specification/alertsmanagement/ @ofirmanor @olalavi @erangon @orieldar @ilaizi @shakednai1 @khaboasb @orenhor
+
+# PRLabel: %API Management
 /specification/apimanagement/ @promoisha @solankisamir
```
Comment (weshaggard) [inline CODEOWNERS:20]: @ruowan @akning-ms unrelated to this PR but there are a lot of invalid users in this code owners file that we should figure out how to get fixed. Can you guys please file a clean-up item for tracking this issue.

Comment (ruowan) [reply CODEOWNERS:20]: We create a issue to track cleanup code owner files. https://github.com/Azure/azure-sdk-tools/issues/3374

**Pull request merged**